devtools/generate_batch_config.py

```diff
@@ -55,7 +55,7 @@ def to_config(
                     ],
                     "computeResource": {
                         "cpuMilli": 8000,
-                        "memoryMib": 32 * 1024,
+                        "memoryMib": 60 * 1024,
```

Comment:
I think this actually still comes out to more than 64 GB: 60 * 1024 = 61,440; 64 * 1e9 / 2**20 = 61,035.15625. if you want something that's more meaningful than a magic number we should do something like

```python
mib_per_gb = 1e9 / 2**20
...
"memoryMib": int(64 * mib_per_gb)
```

Reply:
Ayy, okay. Fixed.